cmd/robot/build.go

```diff
@@ -412,3 +418,42 @@ func (v *trackUpdateVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		return nil
 	}, grpc.WithInsecure())
 }
+
+type packageUpdateVerb struct {
+	RobotOptions
+	Description string `help:"A description of the track"`
+	Parent      string `help:"The id of the package that will be the new parent"`
+}
+
+func (v *packageUpdateVerb) Run(ctx context.Context, flags flag.FlagSet) error {
+	return grpcutil.Client(ctx, v.ServerAddress, func(ctx context.Context, conn *grpc.ClientConn) error {
+		b := build.NewRemote(ctx, conn)
+		args := flags.Args()
+		pkg := &build.Package{
+			Information: &build.Information{Description: v.Description},
+			Parent:      v.Parent,
+		}
+		if len(args) == 0 {
+			return log.Err(ctx, nil, "Missing argument, must specify a package to update")
+		}
+		err := b.SearchPackages(ctx, script.MustParse("Id == $").Using("$")(args[0]).Query(), func(ctx context.Context, entry *build.Package) error {
```

Comment:
you can combine this into:
```
if err := ...; err != nil {
  ...
}
```

Reply:
ok done.

Reply:
uhm... not done.

Reply:
When the logic spans multiple lines, like this does - I actually encourage splitting the assignment and condition to separate statements, just for parsabilities sake.

Reply:
OK, good to know, thanks.

Nick, it's already approved, so you can just submit as is 😄

Reply:
ok I changed it back just to make it all match.